### PR TITLE
add issue templates

### PR DESCRIPTION
**Pull request by djschny**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/report-an-issue.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: ''
+labels: template-bug
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what the problem is.
+
+**To Reproduce**
```
Comment (ndwinton) [inline .github/ISSUE_TEMPLATE/report-an-issue.md:13]: A tiny issue - "reproduce" shouldn't be capitalised for consistency with case elsewhere

Comment (djschny) [reply .github/ISSUE_TEMPLATE/report-an-issue.md]: It's been fixed. Ironically, that was taken as-is from the suggested add a template from GitHub:

https://github.com/vmware-tanzu-learning/tko-ref-arch-training/issues/templates/edit

File: .github/ISSUE_TEMPLATE/suggest-an-idea.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Suggest an Idea
+about: Suggest an idea or change for the training
+title: ''
+labels: template-enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (ndwinton) [inline .github/ISSUE_TEMPLATE/suggest-an-idea.md:10]: How about:

**Why would you like this feature?**
If this solves a problem for you please give a clear and concise description of what the problem is, for example, "I'm always frustrated when ..."

Comment (djschny) [reply .github/ISSUE_TEMPLATE/suggest-an-idea.md]: Sounds good, suggestion applied and updated

**Review by ndwinton — APPROVED** [review]

**Pull request merged**